.github/workflows/image-arm-pr.yaml

```diff
@@ -14,6 +15,21 @@ env:
 jobs:
   opensuse:
     uses: ./.github/workflows/reusable-docker-arm-build.yaml
+    permissions:
+      id-token: write  # OIDC support
+      contents: write
+      security-events: write
+      actions: read
```

Comment:
Do we need to be explicit with the read ones? I would expect this entries to be read by default and only need to override the write ones?

Reply:
I first had it without but it didn't work ... maybe there's a way but i'm not sure about the syntax